frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -864,3 +886,33 @@ func runAcloudSetup(execContext ExecContext, artifactsRootDir, artifactsDir, run
 	go run(execContext(context.TODO(), "sudo", "-u", cvdUser, "chmod", "644", runtimeDir+"/cuttlefish_runtime/logs/metrics.log"))
 	go run(execContext(context.TODO(), "sudo", "-u", cvdUser, "chmod", "644", runtimeDir+"/cuttlefish_runtime/logs/modem_simulator.log"))
 }
+
+func sliceItoa(s []uint32) []string {
+	result := make([]string, len(s))
+	for i := range s {
+		result[i] = strconv.Itoa(i)
+	}
+	return result
+}
+
+func nameToNumber(s string) (int, error) {
+	// instance names follow format "cvd-[NUMBER]".
+	parts := strings.Split(s, "-")
+	if len(parts) != 2 {
+		return 0, fmt.Errorf("failed parsing instance name %q", s)
+	}
+	n, err := strconv.Atoi(parts[1])
+	if err != nil {
+		return 0, fmt.Errorf("failed parsing instance name %q: %w", s, err)
+	}
+	return n, nil
+}
+
+func contains(s []uint32, e uint32) bool {
```

Comment:
nit: this could be generic

Reply:
agree, however in host orchestrator we are contrained to `go 1.13`